### PR TITLE
Create changelog v8.1.0

### PR DESCRIPTION
**Pull request by dfajmon**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Update changelog v8.1.0

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Review by xing-yang — COMMENTED** [review]
A few PRs got merged. Can you generate release-notes again?

File: CHANGELOG/CHANGELOG-9.0.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+### Uncategorized
+
+- Cleanup: Update csi release tools ([#1137](https://github.com/kubernetes-csi/external-snapshotter/pull/1137), [@andyzhangx](https://github.com/andyzhangx))
+- It exposes the metrics for volumegroupsnapshot operations on passing flags like --http-endpoint and --metrics-path to the snapshot controller. These are the command line arguments for the users to enable metrics and start the sever.
+  The metric `snapshot_controller_operation_total_seconds` has been added for the volumegroupsnapshot operations:
+  1. CreateGroupSnapshot
+  2. DeleteGroupSnapshot
+  3. CreateSnapshotAndReady ([#1107](https://github.com/kubernetes-csi/external-snapshotter/pull/1107), [@yati1998](https://github.com/yati1998))
+- Prints additional details when using kubectl get on VolumeSnapshot, VolumeSnapshotContent, and VolumeSnapshotClass API objects. ([#256](https://github.com/kubernetes-csi/external-snapshotter/pull/256), [@huffmanca](https://github.com/huffmanca))
```
[the remark this answers is not in the document]
Comment (dfajmon) [reply CHANGELOG/CHANGELOG-9.0.md]: Not sure either might be a bug in release-notes?
I ran it like this
`release-notes --org=kubernetes-csi --repo=external-snapshotter --required-author="" --markdown-links --output out.md --debug --start-rev=v8.0.0 --end-rev=master`

**Comment by dfajmon** [comment]
@xing-yang updated, thanks for the review

**Pull request merged**